test/unit-tests/textBox.test.js

```diff
@@ -408,7 +408,7 @@ describe(test_name, () => {
                         </div>
                         <div name="withWrappedLabel">
                             <label>
-                                <input type="${inputType.type}"/>
+                            <input type="${inputType.type}" placeholder='Filter'/>
```

Comment:
Can we have a separate input element with only the placeholder for the test?  I notice that this input is used for wrapped label test.